docs/reference/ssl/swarm-with-traefik.md

```diff
@@ -0,0 +1,319 @@
+
+Add the following volumes directive on the last line of the file:
+
+```yaml
+...
+volumes:
+    acme:
+```
+
+## Install OpenFaaS
+
+With all of the modifications in place, you can now deploy using the standard deployment scripts. On Mac and Linux: `deploy_stack.sh` and on Windows use `deploy_stack.ps1`. This script will deploy all of the required resources (services, configuration files, networks, and secrets) for your OpenFaaS cluster as they are defined in the `docker-compose.yaml` file.
+
+First, make sure that Docker will execute commands on your Swarm manager node:
+
+```bash
+eval $(docker-machine env <name-of-manager-node>)
+```
+
+Then, run your deployment script. A successful install will look like this:
+
+```bash
+$ ./deploy_stack.sh
+Attempting to create credentials for gateway..
+adadsf809adf098adfajlk12g
+mb1jl213nlkmqfadsaokv68lh
+[Credentials]
+ username: admin
+ password: <randomly-generated-password>
+ echo -n <randomly-generated-password> | faas-cli login --username=admin --password-stdin
+
+Enabling basic authentication for gateway..
+
+Deploying OpenFaaS core services
+Creating network func_functions
+Creating config func_alertmanager_config
+Creating config func_prometheus_config
+Creating config func_prometheus_rules
+Creating service func_alertmanager
+Creating service func_traefik
+Creating service func_gateway
+Creating service func_faas-swarm
+Creating service func_nats
+Creating service func_queue-worker
+Creating service func_prometheus
+```
+
+## Deploy and Invoke a function
+
+In your projects containing OpenFaaS functions, you can now deploy using your domain as the gateway.
+
+### Deploy from the CLI
+```sh
+faas-cli login --gateway https://openfaas.mydomain.com --username admin --password <randomly-generated-password>
+faas-cli deploy --gateway https://openfaas.mydomain.com
+```
+Replace `openfaas.mydomain.com` with your domain as well as adding the username `admin` and secure random password that the deploy script created for you when you deployed OpenFaaS.
+
+### Using the web UI
+You can use the web UI to see the functions deployed to your cluster or to deploy functions from the Store.  In your web browser, go to https://openfaas.mydomain.com/ui/. Note that the trailing slash is required.
+
+On your first visit, the HTTP authentication dialogue box will open, you can login with the username `admin` and secure random password that the deploy script created for you when you deployed OpenFaaS.
+
+## Verify and Debug
+
+- If you want to tail the Traefik logs, you can use
+```sh
+$ docker service ls -f traefik
+```
+You can see internet traffic logs as well as logs related to the Let's Encrypt certificate process.
+
+## Profit!
```

Comment:
I haven't seen profit used like this before?

Reply:
It was something silly that I had added and quietly made it into the ssl for k8s docs, so I duplicated it here.  I can remove both

Reply:
I think this type of thing sounds better in personal blog posts where the reader is connecting with the writer.